### PR TITLE
Add Travis, update composer and tests

### PR DESCRIPTION
**Pull request by michaelbausor**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: php
+php:
+  - '5.5'
+  - '5.6'
+sudo: required
+dist: trusty
+#before_script: phpenv config-add php-custom-config.ini
```
Comment (garrettjonesgoogle) [inline .travis.yml]: why is this commented out?

Comment (michaelbausor) [inline .travis.yml]: Left over from when I originally was trying to figure out how to make travis work, will remove

**Comment by michaelbausor** [comment]
PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**